samples/msal-node-samples/on-behalf-of/index.js

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const express = require("express");
+const msal = require("@azure/msal-node");
+const http = require("http");
+const https = require("https");
+const jwt = require("jsonwebtoken");
+const jwksClient = require("jwks-rsa");
+require("dotenv").config({ path: "../../../.env" });
+
+/**
+ * Command line arguments can be used to configure:
+ * - The port the application runs on
+ * - The cache file location
+ * - The authentication scenario/configuration file name
+ */
+const argv = require("../cliArgs");
+
+const TEST_CACHE_LOCATION = argv.c || "./data/cache.json";
+const cachePlugin = require("../cachePlugin")(TEST_CACHE_LOCATION);
+
+const webAppConfig = require("./config/WEB-APP.json");
+const webApiConfig = require("./config/WEB-API.json");
+
+if (process.env.OBO_WEB_APP_CLIENT_ID) {
+    webAppConfig.authOptions.clientId = process.env.OBO_WEB_APP_CLIENT_ID;
+}
+if (process.env.OBO_WEB_APP_CLIENT_SECRET) {
+    webAppConfig.authOptions.clientSecret = process.env.OBO_WEB_APP_CLIENT_SECRET;
+}
+
+if (process.env.OBO_WEB_API_CLIENT_ID) {
+    webApiConfig.authOptions.clientId = process.env.OBO_WEB_API_CLIENT_ID;
+    webApiConfig.webApiUrl = `api://${process.env.OBO_WEB_API_CLIENT_ID}/access_as_user`;
+}
+if (process.env.OBO_WEB_API_CLIENT_SECRET) {
+    webApiConfig.authOptions.clientSecret = process.env.OBO_WEB_API_CLIENT_SECRET;
+}
+if (process.env.OBO_WEB_API_TENANT_ID) {
+    webApiConfig.authOptions.authority = `https://login.microsoftonline.com/${process.env.OBO_WEB_API_TENANT_ID}`;
+    webApiConfig.discoveryKeysEndpoint = `https://login.microsoftonline.com/${process.env.OBO_WEB_API_TENANT_ID}/discovery/v2.0/keys`;
+}
```

Comment:
probably don't need these here (if someone is manually running the tests, they would setting the params in config/*.json files)